esmvaltool/cmorizers/data/formatters/datasets/esacci_soilmoisture.py

```diff
@@ -0,0 +1,191 @@
+"""ESMValTool CMORizer for ESACCI-SOILMOISTURE data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    ftp://anon-ftp.ceda.ac.uk/neodc/esacci/soil_moisture/data/
+
+Last access
+    20240626
+
+Download and processing instructions
+    Download the data from:
+      daily_files/COMBINED/v08.1/
+      ancillary/v08.1/
+    Put all files under a single directory (no subdirectories with years).
+    in ${RAWOBS}/Tier2/ESACCI-SOILMOISTURE
+Modification history
+    20240626-cammarano_diego: written.
+"""
+
+import glob
+import logging
+import os
+import iris
+from esmvalcore.cmor.fixes import get_time_bounds
+from esmvalcore.preprocessor import concatenate, monthly_statistics
+from cf_units import Unit
+
+from ...utilities import (
+    convert_timeunits,
+    fix_var_metadata,
+    fix_dim_coordnames,
+    fix_bounds,
+    save_variable,
+    set_global_atts,
+    roll_cube_data
+)
+
+logger = logging.getLogger(__name__)
+
+
+def fix_coords_esacci_soilmoisture(cube,
+                                   overwrite_time_bounds=True,
+                                   overwrite_lon_bounds=True,
+                                   overwrite_lat_bounds=True):
```

Comment:
You only use this function once with the argument `overwrite_time_bounds=False)`, so all these optional arguments can be removed and encoded into the function body